macros/common/stage_external_sources.sql

```diff
@@ -34,26 +34,26 @@
     {% endfor %}
     
     {% if sources_to_stage|length == 0 %}
-        {% do dbt_utils.log_info('No external sources selected') %}
+        {% do log('No external sources selected', info = true) %}
     {% endif %}
             
     {% for node in sources_to_stage %}
 
         {% set loop_label = loop.index ~ ' of ' ~ loop.length %}
 
-        {% do dbt_utils.log_info(loop_label ~ ' START external source ' ~ node.schema ~ '.' ~ node.identifier) -%}
+        {% do log(loop_label ~ ' START external source ' ~ node.schema ~ '.' ~ node.identifier, info = true) -%}
         
         {% set run_queue = dbt_external_tables.get_external_build_plan(node) %}
         
-        {% do dbt_utils.log_info(loop_label ~ ' SKIP') if run_queue == [] %}
-        {% set width = 80 %} {# hard code this for now, use PRINTER_WIDTH flag in v1.0+ #}
+        {% do log(loop_label ~ ' SKIP', info = true) if run_queue == [] %}
+        {% set width = flags.PRINTER_WIDTH %}
```

Comment:
This is still causing bugs - why did you remove the hardcoding patch from v0.7.3?

Reply:
@timbrownsf This version of the package is only for dbt version 1.0 and higher, and the issue was only for versions prior to 1.0 - are you using `--no-version-check` in your dbt command? you may want to switch back to 0.7.3 instead.